src/test/skript/tests/syntaxes/structures/StructUsing.sk

```diff
@@ -0,0 +1,11 @@
+using example feature
+using fizz buzz
+using deprecated feature
+
+test "using":
+	assert the current script is using "example feature" with "using feature failed"
+	assert the current script is using "fizz buzz" with "using exact failed"
+	assert the current script is using "fizzbuzz" with "using other pattern failed"
```

Comment:
_I am lazy to test :D_ .. What happens if pattern named `testing` had a parse tag like `testing :(foo|bar)`, would
- Checking for `is using "testing"` match if user used `testing foo` or `testing bar`?
- Checking for `is using "testing foo"` match if user used `testing bar`?

Reply:
So the way this works is it identifies the `is using %string%` *as* an Experiment itself (in exactly the same way as `using <.+>` does!), and then it compares whether the script's set of known experiments contains what you asked for.

So, with your example if you write `using testing foo` and then ask `is using "testing bar"` that's true because they're both patterns of the same experiment.
This means if you register a pattern like `case sensitiv(e|ity)` then it will be matched correctly.

Reply:
Thank you for the explanation :) That's what I expected, but that means parse tags with different a outcomes will not work currently which is totally fine as we can later improve it if we found a use for it but developers can always create a new experiment for that.

Reply:
Yes I expected this, but I decided that you probably should create multiple experiments rather than trying to use one to mean different things, since this is supposed to be an "is present" flag.

Within actual Skript code you *can* detect things by the parse tag/mark if you need to.

Reply:
I see and agree with you on that. We never know how useful some things are sometimes 😄